app/javascript/components/request-workflow-status/data.js

```diff
@@ -43,13 +43,39 @@ const convertDate = (date) => {
   return formattedDate.toString();
 };
 
+// Converts the duration time in ms and returns a string in format: w days x hours y minutes z seconds
+// duration: time in ms
+const convertDuration = (duration) => {
```

Comment:
Is there not a js library that converts to a "human readable" form?

Reply:
moment.js is the library.

Reply:
I will try to rewrite it with moment.js

Reply:
`moment.duration(item.Duration).humanize()` is I think the entire thing.

Reply:
and it has locale support: `moment.duration(item.Duration).locale("en").humanize()`

Reply:
@GilbertCherrie Since some of the things run in sub-second, we might need to tweak the thresholds to humanize to not round to the minute. See https://momentjs.com/docs/#/durations/humanize/